x-pack/plugins/security_solution/public/flyout/entity_details/mocks/index.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { HostMetadataInterface } from '../../../../common/endpoint/types';
+import { EndpointStatus, HostStatus } from '../../../../common/endpoint/types';
+import type { RiskScoreState } from '../../../entity_analytics/api/hooks/use_risk_score';
+import type {
+  HostItem,
+  HostRiskScore,
+  RiskScoreEntity,
+  UserRiskScore,
+} from '../../../../common/search_strategy';
+import { HostPolicyResponseActionStatus, RiskSeverity } from '../../../../common/search_strategy';
+import { RiskCategories } from '../../../../common/entity_analytics/risk_engine';
+import type { ObservedEntityData } from '../shared/components/observed_entity/types';
+
+const userRiskScore: UserRiskScore = {
+  '@timestamp': '626569200000',
+  user: {
+    name: 'test',
+    risk: {
+      rule_risks: [],
+      calculated_score_norm: 70,
+      multipliers: [],
+      calculated_level: RiskSeverity.high,
+      inputs: [
+        {
+          id: '_id',
+          index: '_index',
+          category: RiskCategories.category_1,
+          description: 'Alert from Rule: My rule',
+          risk_score: 30,
+          timestamp: '2021-08-19T18:55:59.000Z',
```

Comment:
i understand this is unrelated to this PR, but I find it odd we're passing a ISO format on a field called `timestamp`.
The `@timestamp` field is unix time, so is the `oldestAlertTimestamp` 

Should this be looked at?

Reply:
I agree that it is weird that we are using two different formats for timestamp on the same mock.
I checked the docs, and ES can deal with both formats, but it returns formated strings by default. Should I convert everything to strings containing formatted dates (e.g. "2015/01/01 12:10:30")?

> Dates will always be rendered as strings, even if they were initially supplied as a long in the JSON `document.
https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html

Maybe @hop-dev or @nicholasberlin can answer this question better.

Reply:
I think you're right, change them all to formatted dates 👍